src/app/services/db.js

```diff
@@ -1,15 +1,15 @@
 const hosturl = process.env.BACKEND_HOST;
 
-export async function loadTicketsForOwner() {
-    const path = `${hosturl}/tickets`;
+export async function loadTicketsForOwner(owner) {
+    const path = `${hosturl}/tickets?owner=${owner}`;
```

Comment:
probably too late to do it, but in general new URL('/tickets?owner=etc', hosturl).href - you avoid issues if hosturl ended with `/`